collecting-parameter/README.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: pattern
+title: Collecting parameter
+folder: collecting-parameter
+permalink: /patterns/collecting-parameter/
+categories: Structural
```

Comment:
Add a tag as mentioned in the Wiki setion.

Reply:
I'd suggest the category `Idiom` and the tag `Decoupling`